Kernel/API/FileSystem/MountSpecificFlags.h

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2022, Liav A. <liavalb@hotmail.co.il>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <AK/Types.h>
+
+struct [[gnu::packed]] mount_specific_flag {
```

Comment:
Any specific reason why this is packed? Looking at the contents it looks like it won't have any padding anyways, and unaligned pointers are only asking for trouble on ARM-64...